components/search_results/search_limits_banner.tsx

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+import styled from 'styled-components';
+import {useIntl} from 'react-intl';
+import {useSelector} from 'react-redux';
+
+import useOpenPricingModal from 'components/common/hooks/useOpenPricingModal';
+import {getCurrentUser} from 'mattermost-redux/selectors/entities/users';
+import {isAdmin} from 'mattermost-redux/utils/user_utils';
+import {isCurrentLicenseCloud, getSubscriptionProduct as selectSubscriptionProduct} from 'mattermost-redux/selectors/entities/cloud';
+import {CloudProducts} from 'utils/constants';
+import useGetUsage from 'components/common/hooks/useGetUsage';
+import {asGBString, fallbackStarterLimits} from 'utils/limits';
+
+const StyledDiv = styled.div`
+width: 100%;
+`;
+
+const StyledA = styled.a`
+color: rgba(var(--denim-button-bg-rgb, 1)) !important;
+`;
+
+const InnerDiv = styled.div`
+display: flex;
+gap: 8px;
```

Comment:
I don't think this shoudl use gap. That is a grid property but this is a flex container.